### DISCUSSION TITLE
제페토 오류 코드 모음집 같은건 없나요?

### DISCUSSION THREAD

**Discussion by kyn320** [naverz/zepeto-studio-kor#505] [5. Q&A]
가이드 문서나 공식 Docs를 찾아봐도 오류 코드에 대한 내용이 하나도 없는데
이를 확인 할수 있는 곳이 있을까요?

없다면 언제쯤 알수 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~
오류코드 모음집은 없지만, 추후 API document 제작시 기능별로 내용을 포함할 예정입니다. 

만약 특정 오류에 관해 오류 코드가 필요하다면 상세히 요청해 주세요~
감사합니다.
